### ISSUE TITLE
Failed to download any results for xxxxx

### ISSUE THREAD

**Issue by GhostdogNeo** [pymedusa/Medusa#3405]
**Branch/Commit: master (v0.1.19) / ca2c5c4c92888f2f588c74c6770cfa12c6db7f40 **
**OS: Windows-10-10.0.15063 **
**What you did: I try to search or manual search South Park S21E09 episode on yggtorrent provider **
**What happened: torrent is not snatched **
**What you expected: I expect torrent file to be snatched (as it exists on yggtorrent website) **
**Logs:**
```
2017-12-04 20:10:25 WARNING  SNATCHQUEUE-MANUALSNATCH-75897 :: [ca2c5c4] Failed to download any results for South.Park.S21E09.VOSTFR.720p.WEB-DL.x264-FTMVHD
2017-12-04 20:10:08 WARNING  SEARCHQUEUE-DAILY-SEARCH :: [ca2c5c4] Failed to download any results for South.Park.S21E09.Super.Hard.PCness.VOSTFR.720p.WEB-DL.h264-EREBOS
```

**Comment by medariox** [comment]
Need to post a full debug log.

**Comment by GhostdogNeo** [comment]
`2017-12-10 18:20:52 WARNING  SEARCHQUEUE-DAILY-SEARCH :: [1a8dc66] Failed to download any results for South.Park.S21E09.VOSTFR.720p.WEB-DL.x264-FTMVHD
2017-12-10 18:20:52 INFO     SEARCHQUEUE-DAILY-SEARCH :: [1a8dc66] Downloading South.Park.S21E09.VOSTFR.720p.WEB-DL.x264-FTMVHD from Yggtorrent at https://ww1.yggtorrent.com/profile/43158-hijohimoto
2017-12-10 18:20:52 INFO     SEARCHQUEUE-DAILY-SEARCH :: [1a8dc66] Downloading South.Park.S21E09.VOSTFR.720p.WEB-DL.x264-FTMVHD with 70 seeders and 2 leechers and size 252.28 MB from Yggtorrent`

**Comment by labrys** [comment]
@GhostdogNeo it looks like the file you snatched wasn't a proper torrent (corrupted?).  Try to manually snatch another and see if you get the same result.

**Comment by neo62248** [comment]
I think it is a problem about downloading torrent files from yggtorrent... No file can be downloaded, trace below : 

2018-06-16 12:02:14 DEBUG    Thread_15 :: [c65ca26] GET URL: https://cdn.pymedusa.com/providers/broken_providers.json [Status: 200]
2018-06-16 12:02:14 DEBUG    Thread_15 :: [c65ca26] User-Agent: Medusa/0.2.5 (Linux; 3.10.102; 2651ca70-7093-11e8-b6e5-0011326fe02f)
2018-06-16 12:02:14 INFO     Thread_15 :: [c65ca26] Broken providers found: [u'']
2018-06-16 12:02:49 INFO     SNATCHQUEUE-MANUALSNATCH-60866 :: [c65ca26] Beginning to manual snatch release: iZombie.S03E01.FRENCH.720p.BluRay.x264-FRATERNiTY.mkv
2018-06-16 12:02:49 INFO     SNATCHQUEUE-MANUALSNATCH-60866 :: [c65ca26] Downloading iZombie.S03E01.FRENCH.720p.BluRay.x264-FRATERNiTY.mkv with 41 seeders and 0 leechers and size 1.79 GB from Yggtorrent
2018-06-16 12:02:49 DEBUG    SNATCHQUEUE-MANUALSNATCH-60866 :: [c65ca26] POST URL: https://yggtorrent.is/user/login [Status: 301]
2018-06-16 12:02:49 DEBUG    SNATCHQUEUE-MANUALSNATCH-60866 :: [c65ca26] User-Agent: Medusa/0.2.5 (Linux; 3.10.102; 2651ca70-7093-11e8-b6e5-0011326fe02f)
2018-06-16 12:02:49 DEBUG    SNATCHQUEUE-MANUALSNATCH-60866 :: [c65ca26] With post data: id=**********&pass=**********
2018-06-16 12:02:50 DEBUG    SNATCHQUEUE-MANUALSNATCH-60866 :: [c65ca26] GET URL: https://ww1.yggtorrent.is/user/login [Status: 200]
2018-06-16 12:02:50 DEBUG    SNATCHQUEUE-MANUALSNATCH-60866 :: [c65ca26] User-Agent: Medusa/0.2.5 (Linux; 3.10.102; 2651ca70-7093-11e8-b6e5-0011326fe02f)
2018-06-16 12:02:50 DEBUG    SNATCHQUEUE-MANUALSNATCH-60866 :: [c65ca26] GET URL: https://yggtorrent.is/ [Status: 301]
2018-06-16 12:02:50 DEBUG    SNATCHQUEUE-MANUALSNATCH-60866 :: [c65ca26] User-Agent: Medusa/0.2.5 (Linux; 3.10.102; 2651ca70-7093-11e8-b6e5-0011326fe02f)
2018-06-16 12:02:50 DEBUG    SNATCHQUEUE-MANUALSNATCH-60866 :: [c65ca26] GET URL: https://ww1.yggtorrent.is/ [Status: 200]
2018-06-16 12:02:50 DEBUG    SNATCHQUEUE-MANUALSNATCH-60866 :: [c65ca26] User-Agent: Medusa/0.2.5 (Linux; 3.10.102; 2651ca70-7093-11e8-b6e5-0011326fe02f)
2018-06-16 12:02:50 INFO     SNATCHQUEUE-MANUALSNATCH-60866 :: [c65ca26] Downloading iZombie.S03E01.FRENCH.720p.BluRay.x264-FRATERNiTY.mkv from Yggtorrent at https://yggtorrent.is/engine/download_torrent?id=228894
2018-06-16 12:02:50 DEBUG    SNATCHQUEUE-MANUALSNATCH-60866 :: [c65ca26] GET URL: https://yggtorrent.is/engine/download_torrent?id=228894 [Status: 301]
2018-06-16 12:02:50 DEBUG    SNATCHQUEUE-MANUALSNATCH-60866 :: [c65ca26] User-Agent: Medusa/0.2.5 (Linux; 3.10.102; 2651ca70-7093-11e8-b6e5-0011326fe02f)
2018-06-16 12:02:50 DEBUG    SNATCHQUEUE-MANUALSNATCH-60866 :: [c65ca26] GET URL: https://ww1.yggtorrent.is/engine/download_torrent?id=228894 [Status: 200]
2018-06-16 12:02:50 DEBUG    SNATCHQUEUE-MANUALSNATCH-60866 :: [c65ca26] User-Agent: Medusa/0.2.5 (Linux; 3.10.102; 2651ca70-7093-11e8-b6e5-0011326fe02f)
2018-06-16 12:02:50 DEBUG    SNATCHQUEUE-MANUALSNATCH-60866 :: [c65ca26] Setting permissions for /volume1/transmission/watch/iZombie.S03E01.FRENCH.720p.BluRay.x264-FRATERNiTY.mkv.torrent to 438 as parent directory has 511
2018-06-16 12:02:50 DEBUG    SNATCHQUEUE-MANUALSNATCH-60866 :: [c65ca26] Failed to validate torrent file: /volume1/transmission/watch/iZombie.S03E01.FRENCH.720p.BluRay.x264-FRATERNiTY.mkv.torrent. Error: not a valid bencoded string
2018-06-16 12:02:50 DEBUG    SNATCHQUEUE-MANUALSNATCH-60866 :: [c65ca26] /volume1/transmission/watch/iZombie.S03E01.FRENCH.720p.BluRay.x264-FRATERNiTY.mkv.torrent is not a valid torrent file
2018-06-16 12:02:50 WARNING  SNATCHQUEUE-MANUALSNATCH-60866 :: [c65ca26] Failed to download any results for iZombie.S03E01.FRENCH.720p.BluRay.x264-FRATERNiTY.mkv

**Comment by Amonbofils** [comment]
same here, all downloads are like this : 

> 2018-06-19 11:49:56 DEBUG    SNATCHQUEUE-MANUALSNATCH-247897 :: [c65ca26] /home/Downloads/Torrents/Homeland.S07E07.FASTSUB.VOSTFR.720p.WEB-DL.x264.DD+5.1-D7C.mkv.torrent is not a valid torrent file

**Comment by neo62248** [comment]
Change your YGGTORRENT url in yggtorrent.py provider. 
From yggtorrent.is to ww1.yggtorrent.is

And verify your login / password in config of provider (in Medusa application).

Then it would do the job.

**Comment by p0psicles** [comment]
You can also just switch to the develop branch. Should be fixed there.

**Comment by sharkykh** [comment]
Fixed with #4384.
